src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -75,6 +76,9 @@ class OrganizationMemberRequestSerializer(serializers.Serializer):
     email = AllowedEmailField(
         max_length=75, required=True, help_text="The email address to send the invitation to."
     )
+    user_id = serializers.CharField(
```

Comment:
I don't recommend passing both user_id and email because there's a chance they don't match and you have to handle that error in a hacky way. What if you get email and in the endpoint fetch user for that email only if super user?